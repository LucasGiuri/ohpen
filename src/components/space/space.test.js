import Space from './space';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

describe('Space', () => {
  const width = '1230px';
  const height = '1254px';

  it('default snapshot render', () => {
    const tree = renderer.create(<Space />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('with double snapshot render', () => {
    const tree = renderer.create(<Space double />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('with half snapshot render', () => {
    const tree = renderer.create(<Space half />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('with vertical snapshot render', () => {
    const tree = renderer.create(<Space vertical />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('with custom width snapshot render', () => {
    const tree = renderer.create(<Space width={width} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('with custom height snapshot render', () => {
    const tree = renderer.create(<Space height={height} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
