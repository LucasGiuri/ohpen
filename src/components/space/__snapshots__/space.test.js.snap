// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Space default snapshot render 1`] = `
<div
  className="sc-bdfBQB kDlxrm"
  height=""
  width=""
/>
`;

exports[`Space with custom height snapshot render 1`] = `
<div
  className="sc-bdfBQB frKbNU"
  height="1254px"
  width=""
/>
`;

exports[`Space with custom width snapshot render 1`] = `
<div
  className="sc-bdfBQB hvxzpm"
  height=""
  width="1230px"
/>
`;

exports[`Space with double snapshot render 1`] = `
<div
  className="sc-bdfBQB jINbmp"
  height=""
  width=""
/>
`;

exports[`Space with half snapshot render 1`] = `
<div
  className="sc-bdfBQB kDmBBq"
  height=""
  width=""
/>
`;

exports[`Space with vertical snapshot render 1`] = `
<div
  className="sc-bdfBQB hOuFBe"
  height=""
  width=""
/>
`;
